moped-editor/src/views/staff/StaffForm.js

```diff
@@ -260,20 +253,19 @@ const StaffForm = ({
             <FormControl variant="outlined" className={classes.formSelect}>
               <InputLabel id="workgroup-label">Workgroup</InputLabel>
               <Controller
-                name={"workgroup_id"}
+                name="workgroup_id"
                 control={control}
-                render={({ onChange, ref, value }) => (
+                render={({ field }) => (
                   <Select
-                    variant="standard"
-                    id="workgroup"
+                    {...field}
+                    variant="outlined"
+                    id="workgroup_id"
                     labelId="workgroup-label"
                     label="Workgroup"
                     disabled={!isUserActive}
                     onChange={(e) => {
-                      onChange(e.target.value ?? "");
+                      field.onChange(e.target.value ?? "");
                     }}
```

Comment:
@mddilley i noticed that the "workgroup" error message doesn't render if you leave the field blank, and i'm pretty sure that's a bug i introduced a few releases ago. if you look  down below in L286 the render logic is checking `errors.workgroup` when it should now be checking `errors.workgroup_id`.

Reply:
I'll make an attempt to fix this in this PR. It doesn't seem like a big lift. 👍

Reply:
@johnclary I fixed this up - wasn't bad at all. I added some more comments about the changes that I pushed today.